### ISSUE TITLE
Configuring IDE0160 code style adds an .editorconfig description referencing the wrong style

### ISSUE THREAD

**Issue by jnm2** [dotnet/roslyn#57666]
**Version Used**: 17.0.0

'Convert' also seems like an odd name for the rule. Are other rules titled 'Convert X'?

![image](https://user-images.githubusercontent.com/8040367/141026759-700f5917-b42f-4947-b2ad-45f3384c514f.png)

**Comment by Youssef1313** [comment]
Agreed. In most (almost all) cases the codefix title should be different than analyzer title. Can't think of a good analyzer title though :(
